debian/copyright

```diff
@@ -0,0 +1,175 @@
+Format: https://www.debian.org/doc/packaging-manuals/copyright-format/1.0/
+Upstream-Name: libkdumpfile
+Upstream-Contact: cloud-init-dev@lists.launchpad.net
```

Comment:
is this contact email correct? it looks like a email for cloud-init, which is unrelated to this repository, right?

Reply:
Good catch! @sravyamks I think that should be `ptesarik@suse.com`